### DISCUSSION TITLE
v1.1.0

### DISCUSSION THREAD

**Discussion by DamianOsipiuk** [DamianOsipiuk/vue-query#11] [General]
feat(devtools): support for Vue2
feat: es5+cjs, es6+esm, umd builds

<hr /><em>This discussion was created from the release <a href='https://github.com/DamianOsipiuk/vue-query/releases/tag/v1.1.0'>v1.1.0</a>.</em>

**Comment by sondh0127** [comment]
Can you give more example, especially with Nuxt. Thanks

**Reply by DamianOsipiuk** [reply]
Hi, I have not tried to run this lib with Nuxt. As it is SSR it might need some additional compatibility features or fixes.
I will try to make it work with Nuxt after stabilizing the Vue2 support and potentially some documentation.

**Reply by DamianOsipiuk** [reply]
@sondh0127 Nuxt support added in https://github.com/DamianOsipiuk/vue-query/releases/tag/v1.6.0
Please let me know if it works as expected.
